### PR TITLE
Fix typos in servlet docs

### PR DESCRIPTION
**Pull request by pavolloffay**

### CODE REVIEW

File: instrumentation/servlet/README.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ at java.base/java.lang.Thread.run(Thread.java:834)
 Everything starts when HTTP request processing reaches the first class from Servlet specification.
 In the example above this is `ApplicationFilterChain.doFilter(ServletRequest, ServletResponse)` method.
 Let us call this first servlet specific method an "entry point".
-This is the main target for `Servlet3Instrumentation` and `Servlet2Instrumentation` instrumenters:
+This is the main target for `Servlet3Instrumentation` and `Servlet2Instrumentation` instruments:
```
Comment (iNikem) [inline instrumentation/servlet/README.md:45]:
```suggestion
This is the main target for `Servlet3Instrumentation` and `Servlet2Instrumentation` instrumentations:
```

**Review by iNikem — APPROVED** [review]

**Comment by pavolloffay** [comment]
@iNikem PR updated

**Review by iNikem — APPROVED** [review]
Please sync with default to fix build

**Comment by pavolloffay** [comment]
Rebased

**Pull request merged**